src/main/java/com/tutti/server/core/member/infrastructure/MemberBehaviorLogRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.tutti.server.core.member.infrastructure;
+
+import com.tutti.server.core.member.domain.Member;
+import com.tutti.server.core.member.domain.MemberBehaviorLog;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface MemberBehaviorLogRepository extends JpaRepository<MemberBehaviorLog, Long> {
+
+    List<MemberBehaviorLog> findByMember(Member member);
```

Comment:
```suggestion
    List<MemberBehaviorLog> findAllByMemberId(Long memberId);
```
MemberBehaviorLog 엔티티가 member_id 필드를 가지고 있으니까,  파라미터를 이렇게 바꾸면 어때요?
그리고 JPA 문법상 모든 엔티티를 찾을 때는 findAll 을 사용하더라구요

Reply:
반영 완료!